website/src/content/docs/next/quick_start.mdx

```diff
@@ -5,37 +5,41 @@ author: [Simon Boissonneault-Robert, Maxwell Ward]
 
 
 
-## Installing Taquito using npm
+## Installing Taquito Using npm
 
-> For quick-start, you may also like to try out our template/boilerplate app [here][boilerplate]
+> If you just want to get started quickly with a new project, you can use our boilerplate [Vue](https://github.com/ecadlabs/taquito-vue-template) or [React](https://github.com/ecadlabs/taquito-react-template) templates.
 
-The following instructions assume you have a project already created, and you have `npm` installed and operable.
+This quick start guide and subsequent documentation assumes the following:
+- You have npm installed
+- You have an npm project already created
+- You have an understanding of async/await patterns in JavaScript
+- You have access to a Tezos wallet (Temple, Kukai, etc.)
 
 ```bash
 npm install @taquito/taquito
 ```
 
-## Import the library in your project
+## Importing the Library
 
-### Import `TezosToolkit` from `@taquito/taquito` and instantiate it
-
-The constructor of the `TezosToolkit` class takes an RPC URL as a parameter. It can be a string or a [RpcClient](rpc_package.md) object. A list of community-run nodes can be accessed [here](rpc_nodes.mdx#list-of-community-run-nodes).
+A `TezosToolkit` is the core of your Taquito setup. The constructor of the `TezosToolkit` class takes an RPC URL as a parameter. It can be a string or a [RpcClient](rpc_package) object. A list of well-known public RPC nodes can be found [here](rpc_nodes).
```

Comment:
From the prebuilt page [RpcClient](rpc_package) and [here](rpc_nodes) url have excessive "/quick_start/" for example https://head.taquito-584.pages.dev/docs/next/quick_start/rpc_package